src/components/ScrumlrLogo/ScrumlrLogo.scss

```diff
@@ -39,7 +39,7 @@
   }
 
   .scrumlr-logo__stop {
-    stop-color: var(--accent-color);
+    stop-color: var(--accent-color--light);
```

Comment:
The logo seems broken.
![image](https://github.com/user-attachments/assets/1dc95704-0429-418b-aac5-2666feb44cfe)

Reply:
Oops I meant #4310 of course!

Reply:
Should be fixed with #4312 now

Reply:
nice